$OpenBSD: patch-main_go,v 1.2 2021/11/30 11:01:31 sthen Exp $

Despite what pledge(2) implies, fattr is needed for chmod,
wpath is not enough

Index: main.go
--- main.go.orig
+++ main.go
@@ -63,7 +63,7 @@ func main() {
 		}
 		defer pprof.StopCPUProfile()
 	}
-	if err := protect.Pledge("stdio rpath wpath cpath tty proc exec"); err != nil {
+	if err := protect.Pledge("stdio rpath wpath cpath fattr tty proc exec"); err != nil {
 		panic(err)
 	}
 	ctx := context.Background()
